### PR TITLE
[DASH-2084] Dashboard login screen shows when you register from website

### PR DESCRIPTION
**Pull request by wpasinski**

### CODE REVIEW

File: src/pages/setup.js [hunk 1]
```diff
@@ -30,7 +34,7 @@ class SetupPage extends Component {
           className="vm-3-b"
           style={{ textAlign: 'center' }}
         >
-          {'We\'re preparing your account, please wait...'}
+          {"We're preparing your account, please wait..."}
```
Comment (patrykkopycinski) [inline src/pages/setup.js]: do we need `{}`?

File: src/pages/setup.js [hunk 2]
```diff
@@ -13,10 +13,14 @@ class SetupPage extends Component {
...
-      SessionActions.fetchInstance(instance.name);
-      router.push(`/instances/${instance.name}/`);
-    });
+    if (SessionStore.getSignUpMode()) {
+      connection.Instance.please().create({ name }).then((instance) => {
+        SessionActions.fetchInstance(instance.name);
+        router.push(`/instances/${instance.name}/`);
+      });
+
+      SessionStore.removeSignUpMode();
```
Comment (patrykkopycinski) [inline src/pages/setup.js]: why it's outside a callback?

Comment (wpasinski) [inline src/pages/setup.js]: to get `false` in `componentDidMount` in Dashboard component, is it ok?

File: src/pages/setup.js (hunk 1 above)
Comment (wpasinski) [inline src/pages/setup.js]: we don't _need_, but there is an apostrophe.

File: src/pages/setup.js (hunk 2 above)
Comment (patrykkopycinski) [inline src/pages/setup.js]: it should be done in the callback after `router.push` to make sure it fires after a first Instance has been created

File: src/pages/setup.js (hunk 1 above)
Comment (patrykkopycinski) [inline src/pages/setup.js]: use backticks instead of double quotes

File: src/pages/setup.js (hunk 2 above)
Comment (patrykkopycinski) [inline src/pages/setup.js]: yes, it doesn't make sense to have it async, add error handling

File: src/utils/RoutesUtil.js [hunk 3]
```diff
@@ -121,7 +123,7 @@ const RoutesUtil = {
 
     if (lastInstanceName) {
       this.isInstanceAvailable(lastInstanceName)
-        .then(replace({ pathname: `/instances/${lastInstanceName}/` }));
+        .then(replace({ pathname: `/instances/${lastInstanceName}/sockets` }));
```
Comment (patrykkopycinski) [inline src/utils/RoutesUtil.js]: `/` at the end of link

**Pull request merged**